### PR TITLE
Added Michael

### PR DESCRIPTION
**Pull request by Harkins12345**
Added my information to the JSON file

### CODE REVIEW

File: gebru/roster.json [hunk 1]
```diff
@@ -81,6 +81,11 @@
     {
       "name": "Dan Willis",
       "github": "WalkingZ3d"
+    },
+    {
+      "name": "Michael Harkins-Meloy",
+      "github": "Harkins12345",
```
Comment (rom-30) [inline gebru/roster.json:87]: I understand this part :)

File: gebru/roster.json [hunk 2]
```diff
@@ -81,6 +81,11 @@
...
+      "myPets": [{"cat": "Dexter"}, {"chickens": ["Shirley", "Hettie"]}]
```
Comment (rom-30) [inline gebru/roster.json:88]: I am not sure what prompted you to add those? :)

**Review by rom-30 — CHANGES_REQUESTED** [review]

File: gebru/roster.json (hunk 2 above)
Comment (Harkins12345) [reply gebru/roster.json:88]: Well technically the hello github tutorial never said what changes I should make specifically, so I decided to try and show my understanding of the JSON format. I can of course remove them if they're not the changes you were looking for

Comment (Harkins12345) [reply gebru/roster.json:88]: Never mind I didn't read it properly

**Review by rom-30 — APPROVED** [review]
Thanks @Harkins12345

**Pull request merged**